.teamcity/src/main/kotlin/configurations/PerformanceTestsPass.kt

```diff
@@ -68,14 +68,17 @@ subprojects/$performanceProjectName/build/performance-test-results.zip
                 .filter { it.testProjects.isNotEmpty() }
                 .joinToString(",") { "%dep.${it.id}.env.BUILD_ID%" }
 
+            val dependencyBaselines = performanceTestProject.performanceTests.first { it.testProjects.isNotEmpty() }.let { "%dep.${it.id}.performance.baselines%" }
```

Comment:
Would it be better to extract the performance baseline from the json files? Obtaining it here from a dependency build seems strange.

Reply:
We don't store the baseline in JSON files now, so that would be a more complex change. I also feels it's a bit weird, but we're already collecting dependency build ids...